impl/src/main/java/org/jboss/cdi/tck/tests/context/request/jaxrs/Foo.java

```diff
@@ -20,13 +20,16 @@
 import javax.annotation.PreDestroy;
 import javax.enterprise.context.RequestScoped;
 import javax.inject.Inject;
+import java.util.concurrent.atomic.AtomicLong;
 
 @RequestScoped
 public class Foo {
 
     @Inject
     ObservingBean observingBean;
 
+    private static AtomicLong atomicLong = new AtomicLong(System.currentTimeMillis());
```

Comment:
Since you are basing the id off `getAndIncremenet()`, you no longer need to even use current time (or `Long` for that matter). You could simply initialize `AtomicInteger` with 0.

Reply:
I know about that but I decided that the actual value does not mater and so a "pseudo-random" value for every run will also do the job.

Reply:
True, that was just a remark